.changeset/big-wasps-fix.md

```diff
@@ -0,0 +1,34 @@
+---
+'@leafygreen-ui/input-option': major
+---
+
+### API changes
+- Renames `selected` prop to `checked` (this is done to avoid confusion with the `aria-selected` attribute, witch is conditionally applied via the `highlighted` prop)
+ - `checked` applies the `aria-checked` attribute
+ - `checked` _does not_ apply any styles. Any "checked" styles must be applied by the consuming component
```

Comment:
If I understand this correctly, `input-option` used to apply checked/selected styles, but it will no longer do that?